tests/test_pybullet_utils.py

```diff
@@ -0,0 +1,157 @@
+"""Test cases for pybullet_utils."""
+
+from typing import Any, Dict
+import numpy as np
+import pybullet as p
+import pytest
+from predicators.src.pybullet_utils import get_kinematic_chain, \
+    inverse_kinematics
+from predicators.src import utils
+from predicators.src.settings import CFG
+
+_PYBULLET_SCENE_IS_SETUP = False
+_PYBULLET_SCENE_ATTRIBUTES: Dict[str, Any] = {}
+
+
+def _setup_pybullet_test_scene():
```

Comment:
Are these the tests we want to mark as expensive?

Reply:
How's their speed?

Reply:
no, these tests are actually fast -- 0.9 seconds for the whole file.